keyboards/h87a/h87a.h

```diff
@@ -0,0 +1,34 @@
+#ifndef H87A_H
+#define H87A_H
+
+#include "quantum.h"
+
+#define KEYMAP( \
```

Comment:
This matrix needs to be fixed.  It's "wrong". 
Give me a bit and I'll fix it, or somebody else will.

Reply:
Also, the `KEYMAP` should be changed to `LAYOUT`, or even a specific layout.